templates/zerver/reset_confirm.html

```diff
@@ -23,13 +23,14 @@ <h1>{{ _('Set a new password.') }}</h1>
                     </div>
                 </div>
 
-                <div class="input-box">
+                <div class="input-box password-div">
                     <label for="id_new_password1" class="">{{ _('Password') }}</label>
                     <input id="id_new_password1" class="required" type="password" name="new_password1"
                       value="{% if form.new_password1.value() %}{{ form.new_password1.value() }}{% endif %}"
                       maxlength="100"
                       data-min-length="{{password_min_length}}"
                       data-min-guesses="{{password_min_guesses}}" autofocus required />
```

Comment:
I don't see an autocomplete=new-password here; is that intentional?

Reply:
I didn't add that property exclusively here since it wasn't present before but since we are amending the the fields, I'll add it here as well.